### PR TITLE
chore: Map Chart Spec

### PR DESCRIPTION
**Pull request by jsartisan**
## Summary by CodeRabbit

- **New Features**
	- Enhanced accuracy in Map Chart Widget tests with updated element selection methods.
	- Included new test case for immediate validation of title changes in the Map Chart settings.
	- Updated data point interactions in tests to reflect current data context.
	- Added `mapChartWidget` locator for improved test functionality.

- **Bug Fixes**
	- Streamlined visibility tests for the Map Chart Widget, removing unnecessary assertions.

- **Chores**
	- Updated the limited tests configuration to include the new Map Chart tests while excluding the Fork Template test.

> [!WARNING]
> Tests have not run on the HEAD 0a1afd33b44a0a20321e587871a428044fb71990 yet
> <hr>Tue, 17 Sep 2024 07:24:36 UTC

### CODE REVIEW

**Review by sagar-qa007 — CHANGES_REQUESTED** [review]

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts [hunk 1]
```diff
@@ -14,13 +14,13 @@ import EditorNavigation, {
...
 const _mapChartPlot = (text: string) =>
-  "//text()[contains(., '" + text + "')]/..";
+  `//*[name()='svg']//*[name()='text' and contains(text(), '${text}')]`;
 
 describe(
   "Map Chart Widget Functionality",
   { tags: ["@tag.Widget", "@tag.Maps", "@tag.Visual"] },
   function () {
-    it("1. Drag and drop a Map Chart widget and verify", function () {
+    it.only("1. Drag and drop a Map Chart widget and verify", function () {
```
[the remark this answers is not in the document]
Comment (jsartisan) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts]: Done.

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts [hunk 2]
```diff
@@ -14,13 +14,13 @@ import EditorNavigation, {
 
 const _mapChartCaption = "text:last-child";
 const _mapChartPlot = (text: string) =>
-  "//text()[contains(., '" + text + "')]/..";
+  `//*[name()='svg']//*[name()='text' and contains(text(), '${text}')]`;
```
[the remark this answers is not in the document]
Comment (sagar-qa007) [reply app/client/cypress/e2e/Regression/ClientSide/Widgets/Others/MapChart_Spec.ts:17]: Let's optimise this then. Discussing with you offline.

**Review by sagar-qa007 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**